build/common.sh

```diff
@@ -100,20 +100,20 @@ kube::build::get_docker_wrapped_binaries() {
         );;
     "arm")
         local targets=(
-          cloud-controller-manager,armel/busybox
-          kube-apiserver,armel/busybox
-          kube-controller-manager,armel/busybox
-          kube-scheduler,armel/busybox
-          kube-aggregator,armel/busybox
+          cloud-controller-manager,arm32v7/busybox
```

Comment:
Most of the images under `test/images` use `arm32v6/busybox` instead of `arm32v7/busybox`. what is the difference? should we be consistent?

Reply:
should we update the various images in test/images that set `arm=arm32v6/busybox` then?

Reply:
> should we update the various images in test/images that set `arm=arm32v6/busybox` then?

+1

@ixdy Suppose to be ok. @luxas any suggestions?

Reply:
I think it should be `arm32v7/busybox`; as that is the support level. alpine doesn't have v7 so we have to step down and use v6, but that is a special case.

Reply:
we should probably update the `test/images` images to use `arm32v7/busybox` instead of `arm32v6/busybox`, but we don't need to do it in this PR.  cc @mkumatag